frontend/src/utils/data.ts

```diff
@@ -62,7 +62,7 @@ export function summaryStatisticsByServiceAreaAndCensus(
   let censusCategoryGroups = CENSUS_MAPPING[censusCategory]
   let statisticsByGroup: StatisticsByGroup = {}
   censusCategoryGroups.forEach(censusGroup => {
-    statisticsByGroup[censusGroup] = summaryStatistics(rps, adequacies, { censusCategory, censusGroup })
```

Comment:
oh dear, we need to get our formatting configs aligned, or else there's just going to be a battle or adding and taking away spaces inside of brackets

Reply:
We've mostly been consistent with spaces @philipdickinson, sorry :)

Reply:
Oops!